spec/services/top_region_service_spec.rb

```diff
@@ -89,7 +89,7 @@ def refresh_views
 
     refresh_views
 
-    service = TopRegionService.new([organization], Period.month(june_1.end_of_month), scope: :facility)
```

Comment:
This was an error w/ the test setup, the actual code passes in the Period and doesn't go to end of month.